ocean_lib/models/v4/test/test_erc20_enterprise.py

```diff
@@ -235,7 +235,7 @@ def test_buy_from_fre_and_order(
     )
     mock_dai_contract.approve(
         spender=erc20_enterprise_token.address,
-        amount=consume_fee_amount * 2,
+        amount=web3.toWei(1000, "ether"),
```

Comment:
why did you modify this? we can set either to 2**256 -1, either to the exact amount.